libs/ui/src/components/NeoNftCard/NeoNftCard.vue

```diff
@@ -17,26 +17,28 @@
           nft.name || '--'
         }}</span>
 
-        <CollectionDetailsPopover
-          :show-delay="collectionPopoverShowDelay"
-          :nft="nft">
-          <template #trigger>
-            <nuxt-link
-              v-if="
-                variant !== 'minimal' &&
-                (nft.collection.name || nft.collection.id)
-              "
-              :title="nft.collectionName || nft.collection.name"
-              :to="`/${prefix}/collection/${nft.collection.id}`"
-              class="is-size-7 nft-info-collection-name">
-              {{ nft.collection.name || '--' }}
-            </nuxt-link>
-          </template>
-        </CollectionDetailsPopover>
+        <span class="is-ellipsis">
+          <CollectionDetailsPopover
+            :show-delay="collectionPopoverShowDelay"
+            :nft="nft">
+            <template #trigger>
+              <nuxt-link
+                v-if="
+                  variant !== 'minimal' &&
+                  (nft.collection.name || nft.collection.id)
+                "
+                :title="nft.collectionName || nft.collection.name"
+                :to="`/${prefix}/collection/${nft.collection.id}`"
+                class="is-size-7 nft-info-collection-name">
+                {{ nft.collection.name || '--' }}
+              </nuxt-link>
+            </template>
+          </CollectionDetailsPopover>
+        </span>
```

Comment:
```suggestion
        <CollectionDetailsPopover
          :show-delay="collectionPopoverShowDelay"
          :nft="nft"
          class="is-ellipsis"
        >
          <template #trigger>
            <nuxt-link
              v-if="
                variant !== 'minimal' &&
                (nft.collection.name || nft.collection.id)
              "
              :title="nft.collectionName || nft.collection.name"
              :to="`/${prefix}/collection/${nft.collection.id}`"
              class="is-size-7 nft-info-collection-name">
              {{ nft.collection.name || '--' }}
            </nuxt-link>
          </template>
        </CollectionDetailsPopover>
```
no need to wrap it with `<span>`